pkgs/by-name/co/concord/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  curl,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "concord";
+  version = "2.2.1";
+
+  src = fetchFromGitHub {
+    owner = "Cogmasters";
+    repo = "concord";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-8k/W6007U1/s3vx03i1929a5RKZtpW/jOr4JDwmzwp8=";
+  };
+
+  makeFlags = [ "PREFIX=${placeholder "out"}"];
```

nit: tiny formatting difference when I run `nixfmt` locally. Otherwise looks great!

```suggestion
  makeFlags = [ "PREFIX=${placeholder "out"}" ];
```

---

Is that missing space worth fixing in the PR and then waiting for ofborg or is that fine without fixing it @djacu ?

---

I know it was super nitty but a lot of people have been working hard to reformat all of nixpkgs with the new official formatter. There might come a time in the future where a formatting check is added and the PR won't merge unless it passes. So it's good to get in the habit of making sure you run the official formatter on your nix files. :)

Thanks for making the changes and the PR. Looks great!